src/main/java/net/glowstone/block/itemtype/ItemWrittenBook.java

```diff
@@ -20,19 +20,19 @@ public Context getContext() {
 
     @Override
     public void rightClickAir(GlowPlayer player, ItemStack holding) {
-        openBook(player);
+        openBook(player, EquipmentSlot.HAND);
```

Comment:
I was still able to right click with my offhand in air. If this is due to a larger incomplete implementation of hand/offhand usage in Glowstone, please add a TODO.